renderer/components/dashboard/RunTestCard.js

```diff
@@ -26,7 +26,7 @@ const CardDescription = styled(Box)`
 
 const RunTestCard = ({ name, color, icon, description, onClick }) => {
   return (
-    <Card mb={4} p={2} onClick={onClick} data-test-id='card'>
+    <Card mb={4} p={2} onClick={onClick} data-test-id='card' data-testid='card'>
```

Comment:
We can get rid of `data-test-id` and just use `data-testid` in the integration tests also.